frontend/README.md

```diff
@@ -15,7 +15,9 @@ yarn run start
 
 ## Run with a heroku-hosted instance of unleash-api
 
-Alternatively, instead of running unleash-api on localhost, use a remote instance:
+<!-- todo: update this after updating the package.json -->
```

Comment:
The frontend/package.json file has a "start:heroku" command that we should probably update. Once we do, we'll also have to rework this part of the readme, I reckon.